Bug/ con 2960

sijeesh opened the pull request:
Added a new flag "DISABLE_NODE_CONFIGURATION" to disable the node configuration(like package installation,multipath, iscsi,etc.) completely.

datamattsson reviewed (commented):
I'm not loving the inconsistency in the expressions we've bestowed upon us in this file.

```
if [ "$arg" = "--node-service" ]; then
if [ "$nodeservice" = true ]; then
if [ "${DISABLE_NODE_CONFORMANCE}" = "true" ]; then
```

Can we also change `configureNode` to `configurenode` for consistency?

sijeesh commented on cmd/csi-driver/csi-driver.go:
```diff
@@ -116,7 +116,7 @@ func csiCliHandler(cmd *cobra.Command) error {
 	}
 
-	if nodeService {
+	if nodeService && os.Getenv("DISABLE_NODE_CONFIGURATION") == "false" {
```
Planning to set a default value for this flag as false. Updated the code to improve readability and consider values other than "true" as false.

sijeesh commented:
> I'm not loving the inconsistency in the expressions we've bestowed upon us in this file.
> 
> ```
> if [ "$arg" = "--node-service" ]; then
> if [ "$nodeservice" = true ]; then
> if [ "${DISABLE_NODE_CONFORMANCE}" = "true" ]; then
> ```
> 
> Can we also change `configureNode` to `configurenode` for consistency?
Updated entry script to have only camel case variables. Also moved environment variables top to improve readability.

datamattsson reviewed (approved):
LGTM
